README.md

```diff
@@ -253,7 +253,10 @@ Applies a Meta Configuration MOF file to the computer. Similar to the Windows Po
 
 **StartDscConfiguration.py** 
 Applies a configuration MOF file to the computer. Similar to the Windows PowerShell cmdlet: StartDscConfiguration. Requires the path to the configuration MOF to apply. 
-`sudo ./StartDscConfiguration.py –configurationmof /tmp/localhost.mof`
+`sudo ./StartDscConfiguration.py –-configurationmof /tmp/localhost.mof`
+
+You can also supply the force parameter to forcibly remove any current pending configuration before applying the new configuration.
```

Comment:
>current pending configuration [](start = 63, length = 29)

may be remove 'current' word here it gives impression we remove current configuration with --force parameter as well.